libcodechecker/analyze/analyzer.py

```diff
@@ -69,97 +68,44 @@ def _get_skip_handler(args):
-        context.run_id = connection.add_checker_run(' '.join(sys.argv),
-                                                    args.name,
-                                                    context.version,
-                                                    args.force)
+    metadata['checkers'] = {}
+    for analyzer in analyzers:
+        metadata['checkers'][analyzer] = []
 
-        # Clean previous suppress information.
-        client.clean_suppress(connection, context.run_id)
+        for check, data in config_map[analyzer].checks().iteritems():
+            enabled, descr = data
+            if not enabled:
+                continue
+            metadata['checkers'][analyzer].append(check)
 
-        if os.path.exists(suppress_file):
-            client.send_suppress(context.run_id, connection, suppress_file)
-
-        analyzer_config_map = analyzer_types. \
-            build_config_handlers(args,
-                                  context,
-                                  enabled_analyzers,
-                                  connection)
-
-        if skip_handler:
-            connection.add_skip_paths(context.run_id,
-                                      skip_handler.get_skiplist())
-
-    __print_analyzer_version(context, analyzer_config_map)
-
-    LOG.info("Static analysis is starting ...")
+    # Run analysis.
+    LOG.info("Starting static analysis ...")
     start_time = time.time()
 
-    analysis_manager.start_workers(args,
-                                   actions,
-                                   context,
-                                   analyzer_config_map,
-                                   skip_handler)
+    analysis_manager.start_workers(actions, context, config_map,
+                                   args.jobs, args.output_path,
+                                   __get_skip_handler(args), metadata)
 
     end_time = time.time()
-
-    with client.get_connection() as connection:
-        connection.finish_checker_run(context.run_id)
-
     LOG.info("Analysis length: " + str(end_time - start_time) + " sec.")
 
-
-def run_quick_check(args,
-                    context,
-                    actions):
-    """
-    This function implements the "quickcheck" feature.
-    No result is stored to a database.
-    """
-
-    enabled_analyzers, _ = analyzer_types. \
-        check_supported_analyzers(args.analyzers, context)
-
-    actions = prepare_actions(actions, enabled_analyzers)
-
-    analyzer_config_map = \
-        analyzer_types.build_config_handlers(args,
-                                             context,
-                                             enabled_analyzers)
-
-    __print_analyzer_version(context, analyzer_config_map)
-
-    LOG.info("Static analysis is starting ...")
-
-    analysis_manager.start_workers(args, actions, context, analyzer_config_map,
-                                   _get_skip_handler(args), False)
+    metadata['duration'] = end_time - start_time
```

Comment:
We should store the start and the end time separately. The new db schema will store them separately.